plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -190,6 +190,17 @@ SecRule REQUEST_FILENAME "@endsWith /index.php" \
         "t:none,\
         ctl:ruleRemoveById=920450"
 
+# Gutenberg full site editor (v6.3.1+).
+# Requests can contain CSS data, which are detected by libinjection.
+SecRule ARGS_GET "@rx /wp/v[0-9]/global-styles/" \
+    "id:9507145,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ctl:ruleRemoveTargetById=942100;ARGS,\
+    ver:'wordpress-rule-exclusions-plugin/1.0.0'"
+
```

Comment:
Rule shoud look like this:
```
SecRule REQUEST_FILENAME "@endsWith /index.php" \
    "id:9507145,\
    phase:2,\
    pass,\
    t:none,\
    nolog,\
    ver:'wordpress-rule-exclusions-plugin/1.0.1',\
    chain"
    SecRule &ARGS:rest_route "@eq 1" \
        "t:none,\
        nolog,\
        chain"
        SecRule ARGS:rest_route "@rx ^/wp/v[0-9]+/global-styles/[0-9]+$" \
            "t:none,\
            ctl:ruleRemoveTargetById=942100;ARGS"
```

Reply:
Why do you think that?
- first rule for performance and narrowed scope?
- second rule for ?

Reply:
Your rule is invalid, just look at the first line - you are matching **any** GET argument. It is also not tied to anything other so, in fact, you simply disabling rule 942100 for the whole site (conditions you set in the rule can attacker simply create with every request). Exclusion rules must be at tight as possible to not open more holes in the FW as is needed.

Reply:
Yes, understood. I still don't see why you propose to use two chained rules. Wouldn't this do?

```
SecRule REQUEST_FILENAME "@endsWith /index.php" \
    "id:9507145,\
    phase:2,\
    pass,\
    t:none,\
    nolog,\
    ver:'wordpress-rule-exclusions-plugin/1.0.1',\
    chain"
    SecRule ARGS:rest_route "@rx ^/wp/v[0-9]+/global-styles/[0-9]+$" \
            "t:none,\
            ctl:ruleRemoveTargetById=942100;ARGS"
```

Reply:
No because it can be possibly bypassed (depends on the application) by passing two ARGS with the same name. You have to also check the number of arguments.

Reply:
Ouch! But then your version doesn't work either. We would have to check *all* `rest_route` arguments, not just the first one... How would we do that? We could add a rule that blocks any request with more than 1 occurrence of `rest_route`...

Reply:
My rule will prevents it with this:
`SecRule &ARGS:rest_route "@eq 1" \`

Reply:
i.e. the exclusion is NOT activated for requests with multiple `rest_route` arguments.